packages/plugin-ext/src/main/browser/plugin-contribution-handler.ts

```diff
@@ -164,8 +164,8 @@ export class PluginContributionHandler {
                 const darkIconUrl = typeof iconUrl === 'object' ? iconUrl.dark : iconUrl;
                 const lightIconUrl = typeof iconUrl === 'object' ? iconUrl.light : iconUrl;
                 this.style.insertRule('.' + iconClass, theme => `
-                    width: 16px;
-                    height: 16px;
+                    width: 22px;
```

Comment:
How is it related to this PR? Did you check how icons look like in editor toolbars and so on?

Reply:
reverted, moved to this solution specific css